src/main/java/core/basesyntax/ColorSupplier.java

```diff
@@ -1,7 +1,10 @@
 package core.basesyntax;
 
+import java.util.Random;
+
 public class ColorSupplier {
     public String getRandomColor() {
-        return null;
+        int index = new Random().nextInt(Colour.values().length);
```

Comment:
you need to declare a field in this class to hold an instance of `Random`:
```
private final Random random = new Random();
```

this way you won't have to create new `Random` every time this method is called